code/modules/mob/living/simple_animal/abnormality/waw/snow_whites_apple.dm

```diff
@@ -8,6 +8,7 @@ GLOBAL_LIST_EMPTY(vine_list)
 	icon_living = "snowwhitesapple_inert"
 	icon_dead = "snowwhitesapple_dead"
 	var/icon_aggro = "snowwhitesapple_active"
+	mob_biotypes = NONE //there is no plant biotype
```

Comment:
Obsolete variable not used anywhere most of the time.

Reply:
It looks like its put on most monsters though. Its mostly used for logic that requires a creatures general anatomy but snow whites apple is a organic so i guess theres no point 
![image](https://user-images.githubusercontent.com/109536843/211132008-87cb6dc8-6152-4530-ae2a-038ceb135649.png)